setup.py

```diff
@@ -13,6 +13,7 @@
 ]
 
 DEV_REQUIRES = [
+    'pre-commit',
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously. -->
**🎯 suggestion:** I think this is maybe a `TEST` dependency more so? I think the `DEV` onces are intended for building the library rather than testing the library

Reply:
Thanks for addressing 👍